.changeset/little-lizards-brake.md

```diff
@@ -0,0 +1,5 @@
+---
+"@replexica/cli": minor
```

Comment:
I know this isn't ideal, but when the plan is to bump the cli, please bump both `replexica` and `@replexica/cli` packages, and make sure the versions are in sync